yeat/cli/config.py

```diff
@@ -13,15 +13,15 @@
 
 
 PAIRED = ("spades", "megahit", "unicycler")
-PACBIO = ("canu", "flye")
+PACBIO = ("canu", "flye", "hifiasm", "hifiasm-meta")
 OXFORD = ("canu", "flye")
 ALGORITHMS = set(PAIRED + PACBIO + OXFORD)
 
-SHORT_READS = ("paired",)
+ILLUMINA_READS = ("paired",)
 PACBIO_READS = ("pacbio-raw", "pacbio-corr", "pacbio-hifi")
 OXFORD_READS = ("nano-raw", "nano-corr", "nano-hq")
 LONG_READS = PACBIO_READS + OXFORD_READS
-READ_TYPES = SHORT_READS + LONG_READS
+READ_TYPES = ILLUMINA_READS + LONG_READS
```

Comment:
Applied in this suggestion made in https://github.com/bioforensics/yeat/pull/47#discussion_r1267200814 .